draft-packages/tooltip/KaizenDraft/Tooltip/Tooltip.tsx

```diff
@@ -1,64 +1,154 @@
-import * as React from "react"
+import { usePopper } from "react-popper"
+import React, { useState } from "react"
 import classnames from "classnames"
 import styles from "./Tooltip.scss"
+import AppearanceAnim from "./AppearanceAnim"
+import { useUuid } from "./useUuid"
 
 type Position = "above" | "below"
 
 export type TooltipProps = {
+  /**
+   * Use `display="inline"` instead
+   * @deprecated
+   */
   inline?: boolean
+  /**
+   * Unfortunately, the content needed to be wrapped in a div. This can sometimes
+   * break the css layout. To get around this, we allow you to specify the css
+   * display value directly. If you need to need to modify more values, feel free
+   * to use the `classNameAndIHaveSpokenToDST` prop, but avoid it if you can.
+   */
+  display?: "block" | "inline" | "inline-block" | "flex" | "inline-flex"
+  /**
+   * This is more a "desired position". The tooltip will automatically change
+   * its position, if there's not enough room to show it in the one specified.
+   */
   position?: Position
   text: React.ReactNode
   children?: React.ReactNode
   classNameAndIHaveSpokenToDST?: string
+  portalSelector?: string
```

Comment:
Is `portalSelector` used anywhere?

Reply:
Sorry, no it isn't. I forgot to remove this